data/author/sawyer-x.json

```diff
@@ -0,0 +1,7 @@
+{
+   "image" : "/images/author/sawyer-x.jpg",
+   "key" : "sawyer-x",
+   "bio" : "Sawyer X is the Perl-5-Porter's Pumpking, and a core developer of the Dancer [web framework](https://metacpan.org/pod/Dancer2). He's based in Amsterdam, where he works for [booking.com](https://booking.com).",
```

Comment:
"Perl 5 Pumpking" instead of "Perl 5 Porter's Pumpking". The Dancer work is, IMHO, irrelevant here. It could also be "Perl 5 project lead ('Pumpking')". I think the title means little if you're not "in the know" and it's better to clarify the role and the title in parenthesis.